.github/workflows/shinyapps_deploy.yml

```diff
@@ -0,0 +1,70 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+
+name: shiny-apps-deploy
+
+on:
+  push:
+    branches:
+      - master
+      - shinyapps-and-docker
+      - release*
```

Comment:
Should we remove the 'shinyapps-and-docker' branch from here?

Reply:
Can you remove after you merge?  Keeping it in for the moment allows me to see that the build works on my own fork.  Once you merge and then deploy, feel free to remove this line.